src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp

```diff
@@ -53,16 +55,22 @@ namespace argos
       m_cVelocityPrev.Set(0.0, 0.0, 0.0);
       /* reset the drone's angular velocity */
       m_cAngularVelocity.Set(0.0, 0.0, 0.0);
-      m_cAngularVelocityPrev.Set(0.0, 0.0, 0.0);
+      m_cPrevAngularVelocity.Set(0.0, 0.0, 0.0);
       /* reset the drone's previous acceleration */
-      m_cAccelerationPrev.Set(0.0, 0.0, 0.0);
+      m_cPrevAcceleration.Set(0.0, 0.0, 0.0);
       /* reset the drone's previous angular acceleration */
-      m_cAngularAccelerationPrev.Set(0.0, 0.0, 0.0);
-      /* TODO verify if the following variables are necessary */
-      m_cOrientationTargetPrev.Set(0.0, 0.0, 0.0);
-      m_cAngularVelocityCumulativeError.Set(0.0, 0.0, 0.0);
+      m_cPrevAngularAcceleration.Set(0.0, 0.0, 0.0);
+      /* reset the variables for PID controller  */
+      m_cPrevOrientationTarget.Set(0.0, 0.0, 0.0);
+      m_cAngularVelCumulativeError.Set(0.0, 0.0, 0.0);
       m_fAltitudeCumulativeError = 0.0;
-      m_fTargetPositionZPrev = 0.0;
+      m_fPrevTargetAltitude = 0.0;
+      /* reset the gyro bias */   
+      m_fGyroBias = GYRO_BIAS;
+      m_fAccelBias = ACCEL_BIAS;   
+      m_fARW = RW_GYRO;
```

Comment:
I think we can have more readable names for these variables, e.g., `m_fGyroCumulativeError` and `GYRO_CUMULATIVE_ERROR_INIT`